kyuubi-server/src/main/scala/org/apache/kyuubi/server/ui/JettyServer.scala

```diff
@@ -49,7 +52,10 @@ private[kyuubi] class JettyServer(
       case _ =>
     }
   }
-  def getServerUri: String = connector.getHost + ":" + connector.getLocalPort
+
+  @volatile private var _serverUri: String = _
+  def getServerUri: String =
+    Option(_serverUri).getOrElse(connector.getHost + ":" + connector.getLocalPort)
```

Comment:
can we at least log some warning message in `getOrElse` block, to indicate the Jetty server has not been started?

Reply:
done